Dockerfile

```diff
@@ -1,10 +1,10 @@
-FROM registry.opensource.zalan.do/stups/ubuntu:18.04-15
+FROM registry.opensource.zalan.do/stups/ubuntu
```

Comment:
As far as I remember, there is an open source python3 base image that Zalando provides instead of using ubuntu image. :b